package.json

```diff
@@ -26,7 +26,8 @@
 			"framework",
 			"elements/*",
 			"commander",
-			"sdk"
+			"sdk",
+			"protocol-specs"
```

Comment:
With this change, now all elements used in `protocol-specs` will be linked to local packages. Previously only published packages were used in here. Does that will impact any thing?

Reply:
With only lerna, that was the case, but it seems like if we specify the old version, it is fetching the old version. so unless we increase the dependency version it's ok